generic/secrets/security/detected-azure-storage-key.txt

```diff
@@ -0,0 +1,2 @@
+# ruleid: detected-azure-storage-key
+AccountKey=lJzRc1YdHaAA2KCNJJ1tkYwF/+mKK6Ygw0NGe170Xu592euJv2wYUtBlV8z+qnlcNQSnIYVTkLWntUO1F8j8rQ==
```

Comment:
It is helpful and good practice to include test cases that the rule should not detect. These can be annotated with `# ok: detected-azure-storage-key`. With both kinds of tests, we have documented examples of what the rule should and shouldn't detect, and it prevents any regressions from occurring when the rule gets updated, and it helps us make sure the rule only detects what it's supposed to detect.

Could you add some extra test cases for things this should not detect?